docs/notebooks/installation.rst

```diff
@@ -1,77 +1,167 @@
-Installation for Jupyter and JupyterLab
-=======================================
+Installing neptune-notebooks 
+============================
+This page provides instructions for installing neptune-notebooks -- the Neptune Jupyter extension. 
 
-Jupyter
--------
-Install `neptune-notebooks extension <https://github.com/neptune-ai/neptune-notebooks>`_:
+The extension integrates between Neptune and Jupyter or JupyterLab. When the extension is successfully installed, 
+you will be able to upload notebooks to Neptune, check out previously uploaded Notebooks, 
+and track experiments and metrics directly from the Jupyter UI. 
+Neptune versions the Notebook automatically once an experiment has started.
+
+About neptune-notebooks
+^^^^^^^^^^^^^^^^^^^^^^^
+
+neptune-notebooks is a Python package hosted `here <https://pypi.org/project/neptune-notebooks>`_. It includes:
+
+- A very basic CLI (command line interface) for uploading notebooks.
+- Standard Jupyter extension. The extension is written in Javascript and is visible in the web-based Jupyter UI. It is not hosted anywhere separately, only as part of the python package.
+- JupyterLab extension. The extension is written in Javascript and is visible in the web-based Jupyter UI. 
+It is hosted `here <https://www.npmjs.com/package/neptune-notebooks>`_.
+
+
+For Jupyter
```

Comment:
I don't understand this section and even more so the section below "For JupyterLab". What is their purpose?

Reply:
Ok, after reading some more, I now suspect that it's a matter of heading sizes. "For Jupyter" and For "JupyterLab" are actually a very high-level headings.

And "Installation methods" is a subsection of "JupyterLab".
And "Method X:..." are subsections of "Installation methods".

Is that correct? If so, it's not reflected in the menu.

Reply:
You're quite right.
I have changed out the headings so that it makes better sense now.